Resources/Prototypes/_RMC14/Entities/Clothing/Belt/belts.yml

```diff
@@ -286,6 +286,59 @@
       tags:
       - RMCMagazineSmartGun
 
+- type: entity
+  parent: CMBeltMarine
+  id: CMBeltSmartGunOperatorPistol
```

Comment:
This (and all the other pistol belts) should use LimitedStorage component to limit each to one sidearm max, see CMBeltUtilityCombat for an example.
I believe that's how all the "pistol belts" work in 13

Reply:
Oh, that's already taken care of by the holster (or rather the CMItemSlots) component.
The guns are never actually stored in the belt itself, but rather the holster that's attached to the belt.
See short demonstration:

https://github.com/user-attachments/assets/c6db7e22-d7d3-4d54-ac9a-c19067199e14

Any attempt made to insert the gun via clicking just swaps out the gun already in the holster; any attempt made via keybind gives you a prompt telling you there's already something in the holster.
This should be the case for all of the pistol belts I've implemented (I fixed the combat utility belt to use this system as well in #3608).
The only 'holster' that allows for multiple items to be inserted so far is the knife belt (which sets the Count member of CMItemSlots to 12).

Reply:
The belt part itself only allows for ammo (or in the case combat utility belt, ammo and tools) to be inserted, guns are not part of the belt whitelists.

Reply:
That's better for the ergonomics of being able to press E on it but it might get confusing since in 13 you can both quick-unholster and actually see the item in storage
I might try to see if we can do something similar here, using just storage and having activating it unholster the sidearm

Reply:
Alright, I started modifying the holstering system to accommodate storage-based holsters, I'll mark this as a draft until I get it fully working

Reply:
Made the modifications in #3608, the sidearms now get holstered in the storage part of the belt

Reply:
Got the system working in #4522 and updated all of the prototypes here to work with that system